java/core/src/main/resources/rules/microsoft-365/outlook-mail.yaml

```diff
@@ -1,84 +1,86 @@
-allowedEndpointRegexes:
-  - "^/(v1.0|beta)/(groups|users)/?[^/]*"
-  - "^/(v1.0|beta)/(groups|users)\\?.*"
-  - "^/(v1.0|beta)/groups/[^/]*/members[^/]*"
-  - "^/(v1.0|beta)/users/[^/]*/messages/[^/]*"
-  - "^/(v1.0|beta)/users/[^/]*/mailFolders(/SentItems|\\('SentItems'\\))/messages.*"
-  - "^/(v1.0|beta)/users/[^/]*/mailboxSettings"
```

Comment:
main advantage is the inversion of the structure; define endpoints and then transform(s) to apply to each.  

by default, only explicitly allowed endpoints can be called